roles/kubernetes-apps/meta/main.yml

```diff
@@ -36,6 +36,13 @@ dependencies:
       - apps
       - cinder-csi-driver
 
+  - role: kubernetes-apps/csi_driver/aws_ebs
+    when:
+      - aws_ebs_csi_enabled
+    tags:
```

Comment:
You could add the tag "csi-driver" added in this pr: https://github.com/kubernetes-sigs/kubespray/pull/5766
We discussed this a few days ago in another pr

Reply:
I agree